src/redux/reducer.js

```diff
@@ -329,13 +332,26 @@ export const reducer = createReducer(initialState, {
     [CENTER_ON_SUBSTATION]: (state, action) => {
         state.centerOnSubstation = action.centerOnSubstation;
     },
+    [ADD_NOTIFICATION]: (state, action) => {
+        state.notificationList = [
+            ...state.notificationList,
+            action.notification,
+        ];
+    },
+    [REMOVE_NOTIFICATION_BY_NODE]: (state, action) => {
+        state.notificationList = [
+            ...state.notificationList.filter(
+                (n) => n?.nodeUuid !== action.notification.nodeUuid
+            ),
+        ];
+    },
     [OPEN_NETWORK_AREA_DIAGRAM]: (state, action) => {
         state.openNetworkAreaDiagram = action.openNetworkAreaDiagram;
     },
 });
 
 function synchWorkingNodeAndSelectedNode(state) {
-    const workingNode = state.networkModificationTreeModel?.treeElements.find(
+    let workingNode = state.networkModificationTreeModel?.treeElements.find(
```

Comment:
why passing const to let ?

Reply:
i didn't  mean to change it , nice catch ill revert this change